activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -54,7 +54,27 @@ module ActiveSupport
   #
   # Then the messages can be verified and returned upto the expire time.
   # Thereafter, verifying returns +nil+.
+  #
+  # === Rotating keys
+  #
+  # This class also defines a +rotate+ method which can be used to rotate out
+  # no longer used encryption keys. This method accepts the same arguments as
+  # the constructor. This method can be called multiple times and new encryptor
```

Comment:
`constructor` reads like a JavaScript word, I'd say `the same arguments as +new+`.